docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
 
-* Tx fees are calculated based on the **gas_wanted value rather than gas_used**. Therefore users need to be aware of this and should not provide extra gas_wanted values “just in case”.
+    3. The **block proposer** gets a base bonus as 1% of the block rewards
+    * In addition, it gets (4*P)% percent of the block rewards where P=(total power of validators with included precommits / total bonded validator power)
+    * This means it gets an additional bonus between 2.67% and 4%
+    * This means the block proposer’s bonus is between 3.67% and 5% of the total block rewards.
 
-* We have a min-gas-price parameter set at 0.000005 CUDOS below which txs won’t be accepted into mempools
+    4. 20% goes to the community pool.
 
-    This parameter is local to each node on the network, can be changed off-chain, and cannot be queried on-chain.
+    5. The rest is distributed to all of the bonded validators (including the block proposer) proportional to their voting power, regardless of them signing the block.
 
-    Hence we must rely on the community for this implementation to work properly.
 
-* Full-nodes that keep these mempools must have a mechanism to protect themselves from over-usage.
+## Distribution Schedule
 
-:::warning 
-Default `gas-wanted on the CLI is 200,000`. Hence the fee is calculated based on this. Users must be aware of this. Be careful to specify this as what you are prepared to pay. 
+**Tx fees** should be the main source of revenue for **validators** and **delegators**. However, due to the expectation of low early engagement numbers on the **Cudos network**, additional rewards are provided to users. 
 
-Hence most transactions cost 1 CUDOS
-:::
+* A total of ~1,5 billion CUDOS tokens will be distributed as staking rewards in 10 years. The exact number of tokens is given by the formula 
 
-* Sometimes the CLI throws the “Error: RPC error -32603 - Internal error: timed out waiting for tx to be included in a block” error. In this case, the tx is most likely successful, and can be found in the Transactions tab on the explorer page.
+**f(x) = 1.8 x^2 - 53 x + 358** 
 
-One workaround is to use the `--broadcast-mode` flag that returns only a tx hash once a tx is submitted.
+**where *x* is in years and *f(x)* is in millions of CUDOS.**
 
-There was a ticket on making this the default behaviour, but looks like it has been discarded.
+* Hence the amount to be distributed in any given time interval is obtained by integrating the above function over that interval.
 
-Fees on the Keplr extension might be wrong the first time due to some left over data, but this is probably only for the public testnet and NOT for the mainnet.
+* Due to the assumption that tx fees will eventually be the main source of revenue, staking rewards start high and then decrease quadratically in time.
 
-This can be resolved by deleting the chain and reconnecting.
+:::tip Validator commission
```

Comment:
This feels like it's out of place. Maybe best to move it as an additional item in the Block Rewards section?

Reply:
Agreed